internal/config/profile.go

```diff
@@ -37,7 +37,8 @@ import (
 //go:generate mockgen -destination=../mocks/mock_profile.go -package=mocks github.com/mongodb/mongocli/internal/config SetSaver
 
 const (
-	EnvPrefix                    = "mcli"          // EnvPrefix prefix for ENV variables
+	MCLIEnvPrefix                = "MCLI"          // MCLIEnvPrefix prefix for MongoCLI ENV variables
```

Comment:
[nit] `MongoCLIEnvPrefix` to match Atlas pattern?

Reply:
Updated